components/Layout.js

```diff
@@ -17,8 +17,9 @@ export default ({ children }) => {
 
         <meta
           property="og:image"
-          content="http://copenhagenjs.dk/static/images/logo.png"
+          content="http://copenhagenjs.dk/static/images/logo-small.png"
```

Comment:
Why should it be the small one?

Reply:
1. I just updated files name to reflect changes `static/images/logo.png → static/images/logo_small.png `
2. small is 100x100 px and large is 523px, maybe SVG is better?

Reply:
It should be pretty big, https://developers.facebook.com/docs/sharing/best-practices#images